model/openapi.yml

```diff
@@ -126,15 +127,17 @@ components:
         - metadata
       properties:
         type:
+          deprecated: true
```

Comment:
Is there a plan to use different enrollment types or `shared_id`?
cc @cmacknz @blakerouse @michalpristas

Reply:
Not that I know about.